helm-charts/aerospike-cluster/values.yaml

```diff
@@ -16,6 +16,9 @@ image:
 ##   - secret_with_credentials_to_custom_registry
 imagePullSecrets: {}
 
+## Custom labels that will be applied on the aerospikecluster resource
+customLabels: {}
```

Comment:
Please update this field description in the helm-chart README as well

Reply:
Hello Abhisek
I just updated the README of the helm chart.
Thanks
Jean-Francois